android/java/res/layout/rewards_panel_vbat_expire_notice.xml

```diff
@@ -0,0 +1,145 @@
+    android:layout_gravity="center_horizontal"
+    android:gravity="center"
+    android:visibility="gone"
+    app:cardBackgroundColor="@color/rewards_panel_card_bg_color"
+    app:cardCornerRadius="8dp"
+    app:cardElevation="10dp"
+    app:cardUseCompatPadding="true">
+
+    <androidx.core.widget.NestedScrollView
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:fadeScrollbars="false"
+        android:scrollbars="vertical">
+
+        <FrameLayout
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content">
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                android:orientation="vertical">
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:background="@color/vbat_warning_color"
+                    android:orientation="vertical">
+
+                    <ImageView
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_marginStart="24dp"
+                        android:layout_marginTop="24dp"
+                        android:layout_marginBottom="16dp"
+                        android:contentDescription="@null"
+                        app:srcCompat="@drawable/ic_warning_triangle_outline" />
+
+                    <TextView
+                        android:id="@+id/vbat_modal_title"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_marginHorizontal="24dp"
+                        android:layout_marginBottom="24dp"
+                        android:fontFamily="@font/poppins_600"
+                        android:gravity="center"
+                        android:textAlignment="viewStart"
+                        android:textColor="@color/rewards_tipping_text_color"
+                        android:textSize="16sp" />
+
+                </LinearLayout>
+
+                <LinearLayout
+                    android:layout_width="match_parent"
+                    android:layout_height="wrap_content"
+                    android:orientation="vertical">
+
+                    <TextView
+                        android:id="@+id/vbat_modal_text"
+                        android:layout_width="wrap_content"
+                        android:layout_height="wrap_content"
+                        android:layout_gravity="center"
+                        android:layout_marginHorizontal="24dp"
+                        android:layout_marginTop="24dp"
+                        android:fontFamily="@font/poppins_400"
+                        android:gravity="center"
```

Comment:
hope anyone is sufficient  android:layout_gravity="center" and android:gravity="center"  OR android:textAlignment="viewStart"